radix-engine/src/blueprints/consensus_manager/validator.rs

```diff
@@ -1589,13 +1612,13 @@ impl ValidatorBlueprint {
             Decimal::zero()
         } else {
             amount_of_stake_units
-                .safe_div(total_stake_unit_supply)
-                .and_then(|d| d.safe_mul(active_stake_amount))
-                .ok_or_else(|| {
-                    RuntimeError::ApplicationError(ApplicationError::ValidatorError(
+                .safe_mul(active_stake_amount)
```

Comment:
We should do division before multiplication here to reduce chance of overflow as well as be more consistent with pools

Reply:
good catch!

Reply:
Done